src/AppInstallerCLICore/Workflows/UpdateFlow.cpp

```diff
@@ -64,11 +64,18 @@ namespace AppInstaller::CLI::Workflow
 
         // The version keys should have already been sorted by version
         const auto& versionKeys = package->GetAvailableVersionKeys();
+        // Assume that no update versions are applicable
+        bool upgradeVersionApplicable = false;

```

Comment:
```suggestion
        bool upgradeVersionAvailable = false;

```
nit: since we are differentiating applicable vs, applicable